sdks/python/src/opik/cli/imports/experiment.py

```diff
@@ -0,0 +1,541 @@
+            type=experiment_info.get("type", "regular"),
+        )
+
+        # Process experiment items
+        experiment_items = []
+        successful_items = 0
+        skipped_items = 0
+
+        for item_data in items_data:
+            # Handle trace reference (from deduplicated exports)
+            trace_id = handle_trace_reference(item_data)
+            if not trace_id:
+                console.print(
+                    "[yellow]Warning: No trace ID found, skipping item[/yellow]"
+                )
+                skipped_items += 1
+                continue
+
+            # Translate trace id from source (workspace A) to newly created trace id (workspace B)
+            new_trace_id = translate_trace_id(trace_id, trace_id_map)
+            if not new_trace_id:
+                console.print(
+                    f"[yellow]Warning: No mapping for trace {trace_id}. Skipping item.[/yellow]"
+                )
+                skipped_items += 1
+                continue
+
+            # Handle dataset item
+            dataset_item_data = item_data.get("dataset_item_data")
+
+            # If dataset_item_data is missing, try to use input as fallback (for older exports)
+            if not dataset_item_data:
+                # Check if input field exists and use it as dataset item data
+                input_data = item_data.get("input")
+                if input_data and isinstance(input_data, dict):
+                    dataset_item_data = input_data
+                    if debug:
+                        console.print(
+                            "[blue]Using 'input' field as dataset_item_data (export may be from older version)[/blue]"
+                        )
+                else:
+                    console.print(
+                        "[yellow]Warning: No dataset item data found, skipping item. "
+                        "This may be due to an older export format. Please re-export the experiment.[/yellow]"
+                    )
+                    skipped_items += 1
+                    continue
+
+            # Ensure dataset_item_data is a dict
+            if not isinstance(dataset_item_data, dict):
+                console.print(
+                    "[yellow]Warning: dataset_item_data is not a dictionary, skipping item[/yellow]"
+                )
+                skipped_items += 1
+                continue
+
+            try:
+                # Find or create dataset item (prefer deterministic id to avoid extra reads)
+                dataset_item_id = find_dataset_item_by_content(
+                    dataset, dataset_item_data
+                )
+                if not dataset_item_id:
+                    # Prefer using provided id if present, otherwise generate one
+                    provided_id = item_data.get(
+                        "dataset_item_id"
+                    ) or dataset_item_data.get("id")
+
+                    if ds_item is not None:
```

Comment:
`ds_item` is a module, it can't be None. This if-condition is not needed.

Same goes for `id_helpers is not None`

Btw I recommend keeping the module names as is if possible, and rename variables instead if there is a conflict. It will increase the usefulness of keeping the namespace clean by importing module names.